read-the-docs/2/conf.py

```diff
@@ -101,6 +108,7 @@ def parse_major_version(version):
     'github-build-examples': ('https://github.com/axonivy/project-build-examples/blob/' + buildExampleVersion + '/compile-test%s', None),
     'api-browser-url': ('https://developer.axonivy.com/api-browser?configUrl=https://developer.axonivy.com/doc/' + version + '/openapi/config.json&urls.primaryName=%s', None),
     'project-build-plugin-doc': ('https://axonivy.github.io/project-build-plugin/release/%s', None),
+    'link-url': ('https://developer.axonivy.com/link/%s/' + branchVersion, None),
```

Comment:
I think some of them are not needed anymore in master?
But we still want to keep it because we have release/8.0 branch.

Can you please add comments to those which we want to delete someday?